index.html

```diff
@@ -0,0 +1,107 @@
+         or perform file manipulations on a server.
+      </li>
+    </ul>
+    <h2 id="section2">What you should already know</h2>
+    <p>This guide assumes you have the following basic background.</p>
+    <ul>
+      <li>A general understanding of the Internet and the World Wide Web (WWW).</li>
+      <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+      <li>Some programming experience. If you are new to programming, 
+        try one of the tutorials linked on the main page about JavaScript.
+      </li>
+    </ul>
+    <h2 id="section3">JavaScript and Java</h2>
+    <p>
+      JavaScript and Java are similar in some ways but fundamentally different in some others. 
+      The JavaScript language resembles Java but does not have Java's static typing and strong type checking. JavaScript follows most Java expression syntax,
+      naming conventions and basic control-flow constructs which was the reason why it was renamed from LiveScript to JavaScript.
+    </p>
+    <p>
+      In contrast to Java's compile-time system of classes built by declarations,
+      JavaScript supports a runtime system based on a small number of data types representing numeric,
+      Boolean, and string values. JavaScript has a prototype-based object model instead of the more common class-based object model. 
+      The prototype-based model provides dynamic inheritance; 
+      that is, what is inherited can vary for individual objects.
+      JavaScript also supports functions without any special declarative requirements. 
+      Functions can be properties of objects, executing as loosely typed methods.
+    </p>
+    <p>
+      JavaScript is a very free-form language compared to Java.
+      You do not have to declare all variables, classes, and methods.
+      You do not have to be concerned with whether methods are public, private, or protected, 
+      and you do not have to implement interfaces. Variables, parameters, 
+      and function return types are not explicitly typed.
+    </p>
+    </div>
+  </main>
+
+  <aside>
+    <h2 class="main-heading">JS Documentation</h2>
+   <nav>
+     <ul>
+        <li><a href="#section1">Introduction</a></li>
+        <li><a href="#section2">What you should already know</a></li>
+        <li><a href="#section3">JavaScript and Java</a></li>
+        <li><a href="#">Hello World</a></li>
+        <li><a href="#">Variables</a></li>
+        <li><a href="#">Declaring Variables</a></li>
+        <li><a href="#">Variable Scope</a></li>
+        <li><a href="#">Global Variables</a></li>
+        <li><a href="#">Constants</a></li>
+        <li><a href="#">Data Types</a></li>
+        <li><a href="#">If...Else Statements</a></li>
+        <li><a href="#">While Statement</a></li>
+        <li><a href="#">Function Declarations</a></li>
+        <li><a href="#">Reference</a></li>
+        
+     </ul>
+   </nav>
+  </aside>
+</div>
+
+  
+</body>
+</html>
```

Comment:
Please remove empty lines from here as well.

Reply:
@BenMukebo Ben Please assist me.. where am I going wrong.. This project was for my "Review" review.. not codes.